src/carma-1-tenth/run_c1t_metric_analysis.py

```diff
@@ -119,7 +114,7 @@ def test_C1T_12_communicate_load_and_unload(self):
         self.assertTrue(check_port_drayage_ack(self.bag_dir, "PICKUP"), "Vehicle did not ack PICKUP correctly")
         self.assertTrue(check_port_drayage_ack(self.bag_dir, "DROPOFF"), "Vehicle did not ack DROPOFF correctly")
 
-    def test_C1T_13_communicate_inspection(self):
+    def test_C1T_16_communicate_inspection(self):
```

Comment:
test 16?

Reply:
Tests 10, 11, 14, and 15 are visual observations related to the Port Drayage web UI, so we don't have scripts to compute these